src/api/reports/summary_report.py

```diff
@@ -37,16 +38,86 @@
 from ..utils.timer import timing
 
 
+aca_benthic_key, aca_benthic_field = Covariate.SUPPORTED_COVARIATES[0]
+aca_geomorphic_key, aca_geomorphic_field = Covariate.SUPPORTED_COVARIATES[1]
+
+
+def _update_covariate_lookup(covar_lookup, site_id, covariates):
+    covar_lookup[site_id] = ["", ""]
+    for covariate in covariates:
+        values = covariate.value
+        if values is None:
+            continue
+        covar_name = covariate.name
+        if covar_name == aca_benthic_key:
+            values = sorted(values, key=lambda x: (x["area"]), reverse=True)
+            covar_lookup[site_id][0] = values[0]["name"] if values else ""
+        elif covar_name == aca_geomorphic_key:
+            values = sorted(values, key=lambda x: (x["area"]), reverse=True)
+            covar_lookup[site_id][1] = values[0]["name"] if values else ""
+
+
+def _covariate_aca_lookup(site_id_index, content):
+    if isinstance(content, list) is False or len(content) < 2:
+        return None
+
+    site_ids = list({r[site_id_index] for r in content[1:]})
+    sites = Site.objects.filter(id__in=site_ids)
+    covar_lookup = {}
+
+    for s in sites:
+        covariates = s.covariates.filter(name__in=[aca_benthic_key, aca_geomorphic_key])
+        if not covariates:
+            continue
+
+        site_id = str(s.id)
+        covar_lookup[site_id] = ["", ""]
+        _update_covariate_lookup(covar_lookup, site_id, covariates)
+
+    return covar_lookup
+
+
+def _append_aca_covariates(site_id_index, content, view_serializer):
+    """ Append Alan Coral Atlas covariates """
+
+    num_additional_fields = len(view_serializer.additional_fields)
+    column_names = [f.display for f in view_serializer.fields]
+    column_names.extend([aca_benthic_field, aca_geomorphic_field])
+
+    covariate_lookup = _covariate_aca_lookup(site_id_index, content)
+    data = [column_names]
+    for row in content[1:]:
+        site_id = row[site_id_index]
+        covariates = covariate_lookup.get(site_id)
+        row = row[0:-1 * num_additional_fields]
+        row.extend(covariates)
+        data.append(row)
+    
+    return data
+
+
 def get_viewset_csv_content(view_cls, project_pk, request):
-    request = MockRequest(query_params={"field_report": "t"})
+    request = MockRequest()
```

Comment:
Sorry can you remind me why we don't want this param again?

Reply:
Site id is needed to look up the covariates.